src/DataFrames.jl

```diff
@@ -58,6 +59,8 @@ export AbstractDataFrame,
        rownumber,
        select!,
        select,
+       setdisplay_prettytables,
+       setdisplay_traditional,
```

Comment:
I think this is not needed anymore - right? (also can you please review the PR if there are no other "leftovers" after experimentation?)

Reply:
Yes, you are right. I will review to see if there are some trash left from my testing. Sorry about that.